Makefile.am

```diff
@@ -100,7 +100,6 @@ csharp_EXTRA_DIST=                                                           \
   csharp/src/Google.Protobuf.Test/JsonFormatterTest.cs                       \
   csharp/src/Google.Protobuf.Test/JsonParserTest.cs                          \
   csharp/src/Google.Protobuf.Test/JsonTokenizerTest.cs                       \
-  csharp/src/Google.Protobuf.Test/Reflection/CustomOptionsTest.cs            \
```

Comment:
CustomOptionsTest.cs  is removed, does this change breaks current usages for custom options?

Reply:
Yes. I realized it wasn't a good idea a few days ago and have been working on a workaround that lets CustomOptions work through the ExtensionSet.

Reply:
With [the most recent commits](https://github.com/protocolbuffers/protobuf/pull/5350/commits/9084c23dbfabb738c28fb445bc6d62df309898ea) I reimplemented CustomOptions through the ExtensionSet and removed tests that no longer apply to the current implementation (empty set equality and reading values tests).